packages/client/test/end-to-end/dataunion/LoginEndpoints.test.ts

```diff
@@ -1,89 +0,0 @@
-import assert from 'assert'
```

Comment:
Remove also `LoginEndpoints.ts` and `Session.ts` as those are not needed anymore

Reply:
Done